### ISSUE TITLE
Cannot reverse control focus with Shift + Tab key.

### ISSUE THREAD

**Issue by beru** [beru/nana#4]
And I don't know how to set tab-index either..

**Comment by beru** [comment]
By calling these APIs, focus of widgets can be changed by pressing tab key.

``` c++
API::tabstop(widget);
API::eat_tabstop(widget, false);
```

I use following utility function to set tabstop to mutiple widgets.

``` c++
static inline
void set_tabstop(std::initializer_list<widget*> widgets)
{
    for (auto* widget: widgets) {
        API::tabstop(*widget);
        API::eat_tabstop(*widget, false);
    }
}
```

**Comment by beru** [comment]
File : source\gui\detail\win32\bedrock.cpp
Func : nana::detail::Bedrock_WIN32_WindowProc

``` c+++
// case WM_KEYDOWN:
auto the_next = brock.wd_manager.tabstop(msgwnd, (::GetKeyState(VK_SHIFT) >= 0));
```

This modification can solve the issue. :smiley:

**Comment by beru** [comment]
the fix has been merged into official cnjinhao/nana/hotfixes-1.0.1 branch.
cnjinhao has also implemented the feature for X11 platform.
